.storybook/preview.tsx

```diff
@@ -5,17 +5,18 @@ import {MobileProvider, Platform} from '@gravity-ui/uikit';
 import React from 'react';
 import {MINIMAL_VIEWPORTS} from '@storybook/addon-viewport';
 import type {Decorator, Preview} from '@storybook/react';
-import {themeLight} from './theme';
+import {themeLight} from './theme/light';
 import {PageConstructorProvider} from '../src/containers/PageConstructor/Provider';
-import {withTheme} from './decorators/withTheme';
 import {withMobile} from './decorators/withMobile';
 import {withLang} from './decorators/withLang';
 import {DocsDecorator} from './decorators/docs';
 
-import {ThemeProvider} from '../src';
+import {Theme} from '../src';
+
 import {configure, Lang} from '../src/utils/configure';
 
 import '../styles/styles.scss';
+import {GlobalThemeController} from './theme/utils/global-theme-controller';
```

Comment:
please move it before scss import

Reply:
ok.
one should tune `import/order` rule to prevent such mistakes)